lab2.js

```diff
@@ -129,22 +229,46 @@ function lastLetterSort(stringArray) {
     // https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/String
   }
   stringArray.sort(byLastLetter);
+  console.log(stringArray);
 }
+lastLetterSort(stringTest);
+assert(stringTest[0] === 'banana', 'banana needs to go first');
+
+var numberTest = [1, 2, 3, 4, 5];
 
 function sumArray(numberArray) {
   var sum = 0;
-  // TODO: implement me using forEach
+  numberArray.forEach(function(value) {
+    sum += value;
+  });
   return sum;
 }
+sumArray(numberTest);
+assert((sumArray(numberTest)) === 15, 'wrong sum, try again');
+
+var numArr2 = [13, 435, 345, 243, 346, 256546, 2456, 245];
+var numArr3 = [-3, 4, -43, 1.345, 234, 666];
+var leagueOfExtrodianaryArrays = [numArr2, numArr3, numberTest];
```

Comment:
is this like the League of Extraordinary Dancers? Because they are awesome. 
one of my all-time favorite pieces: https://www.youtube.com/watch?v=gNaHaO4KNtY